docker-compose.test.yml

```diff
@@ -225,13 +225,13 @@ services:
     extends: test-gpu-base
     build:
       args:
-        CUDA_DOCKER_VERSION: 11.6.1-devel-ubuntu20.04
-        CUDNN_VERSION: 8.4.1.50-1+cuda11.6
+        CUDA_DOCKER_VERSION: 11.8.0-devel-ubuntu20.04
+        CUDNN_VERSION: 8.6.0.163-1+cuda11.8
         NCCL_VERSION_OVERRIDE: 2.11.4-1+cuda11.6
```

Comment:
should we also move NCCL to cuda 11.8, e.g. [libnccl2_2.15.5-1+cuda11.8_amd64.deb](https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/libnccl2_2.15.5-1+cuda11.8_amd64.deb)?

Reply:
yep, i think we should.

Reply:
hmm looks like the new nccl has some issues, seeing this in the tests: 
`"[1]<stdout>:E     tensorflow.python.framework.errors_impl.UnknownError: {{function_node __wrapped__HorovodAllgather_device_/job:localhost/replica:0/task:0/device:GPU:0}} ncclAllGather failed: invalid argument [Op:HorovodAllgather] name: test_start.sz"`
will have to upgrade it in another separate pr then.